gradle-conjure/src/main/java/com/palantir/gradle/conjure/ConjurePlugin.java

```diff
@@ -634,14 +629,32 @@ static void applyDependencyForIdeTasks(Project project, TaskProvider<?> compileC
             } catch (UnknownTaskException e) {
                 // eclipseClasspath is not always registered
             }
+        });
+    }
+
+    static void configureIdeGeneratedSources(Project project, Provider<Directory> generated) {
+        project.getPlugins().withType(IdeaPlugin.class, plugin -> {
+            IdeaModule module = plugin.getModel().getModule();
+
+            // module.getSourceDirs / getGeneratedSourceDirs could be an immutable set, so defensively copy
+            module.setSourceDirs(mutableSetWithExtraEntry(
```

Comment:
`mutableSetWithExtraEntry` is bringing back memories

Reply:
bad memories I hope

Reply:
yes, how did you guess??